### ISSUE TITLE
Feature Request - Apple App Store support 

### ISSUE THREAD

**Issue by sthristov** [Provenance-Emu/Provenance#2322]
Apple recently opened the App Store for emulators, it would be great if people can install Provenance directly from it.

Reference: [Apple opens the App Store to retro game emulators](https://www.theverge.com/2024/4/5/24122341/apple-app-store-game-emulators-super-apps)

**Comment by FunnyFly-ZW** [comment]
Delta is already there https://apps.apple.com/cn/app/delta-game-emulator/id1048524688

**Comment by SandeMC** [comment]
https://www.imore.com/iphone/exclusive-iphone-emulator-provenance-following-delta-onto-the-app-store-with-sega-and-playstation-support-in-tow

**Comment by gorkemg** [comment]
Please consider releasing Provenance on the App Store also for us in Europe. 
Delta is only available via AltStore PAL, which is only available on iPhone, not any other device category like tvOS or even iPad.

**Comment by axemanozh** [comment]
It's unclear from the iMore article, but this will cover the AppleTV version as well? The Delta developers don't seem particularly interested in an AppleTV version, for whatever reason.

**Comment by esusatyo** [comment]
Yes it does cover Apple TV. There's now quite a few emulators on the store: https://www.macrumors.com/2024/05/16/iphone-app-store-emulators-list/

**Comment by FunnyFly-ZW** [comment]
> Yes it does cover Apple TV. There's now quite a few emulators on the store: https://www.macrumors.com/2024/05/16/iphone-app-store-emulators-list/

Already hands on Retroarch, it runs pretty well.

**Comment by lifeiscontent** [comment]
@JoeMatt any thoughts on this?

**Comment by JoeMatt** [comment]
I've been working on this for the last 5 months. Follow our twitter for updates

**Comment by bradgessler** [comment]
Yes, please release to the App Store, particularly for AppleTV.

For whatever reason, the only AppleTV emulator is RetroArch, and it looks janky.

I'd like something with a file browser interface I could point to an SMB drive so I can load games from my NAS and play them on AppleTV.

**Comment by Chorwacjen** [comment]
since its on the appstore, shouldnt this be closed?

**Comment by eduo** [comment]
I think this ticket can be closed now :D

**Comment by JoeMatt** [comment]
done, ios and tvos (tvos in review)
